faker/providers/python/__init__.py

```diff
@@ -197,7 +197,7 @@ def pydecimal(self, left_digits=None, right_digits=None, positive=False, min_val
                 left_number = "".join([str(self.random_digit()) for i in range(0, left_digits)]) or "0"
         else:
             if min_value is not None:
-                left_number = str(self.random_int(max(max_value or 0, 0), abs(min_value)))
+                left_number = str(self.random_int(max(min_value or 0, 0), abs(max_value)))
```

Comment:
to avoid further confusion, can we also use named arguments?

```suggestion
                left_number = str(self.random_int(min=max(min_value or 0, 0), max=abs(max_value)))
```